packages/terra-form-select/src/clinical-lowlight-theme/Frame.module.scss

```diff
@@ -37,6 +37,8 @@
     --terra-form-select-invalid-focus-background-size: 0 0;
     --terra-form-select-invalid-focus-border-color: #fb4c4c;
     --terra-form-select-invalid-focus-box-shadow: 0 0 0 1px #fb4c4c, 0 0 0 1px #fb4c4c;
+    --terra-form-select-single-select-invalid-focus-border-color: #004c76;
```

Comment:
style values are missing for orion-fusion-theme

Reply:
Above styles has been removed and added outline around the red colour border. Since there is no red colour border for invalid example in fusion theme, outline is not required so i have removed outline for fusion theme.